lib/improver/tests/blending/weighted_blend/test_WeightedBlendAcrossWholeDimension.py

```diff
@@ -172,18 +174,20 @@ def setUp(self):
 
         self.weights1d = Cube(
             weights1d, long_name='weights',
-            dim_coords_and_dims=[(time_coord, 0)],
-            aux_coords_and_dims=[(frt_coord, 0), (fp_coord, 0)])
+            dim_coords_and_dims=[(frt_coord, 0)],
+            aux_coords_and_dims=[(fp_coord, 0)])
+        self.weights1d.add_aux_coord(time_coord)
 
         self.weights_threshold = Cube(
             weights_threshold, long_name='weights',
             dim_coords_and_dims=[(threshold, 0)])
 
         self.weights3d = Cube(
             weights3d, long_name='weights',
-            dim_coords_and_dims=[(time_coord, 0), (lat_coord, 1),
+            dim_coords_and_dims=[(frt_coord, 0), (lat_coord, 1),
                                  (lon_coord, 2)],
-            aux_coords_and_dims=[(frt_coord, 0), (fp_coord, 0)])
+            aux_coords_and_dims=[(fp_coord, 0)])
+        self.weights3d.add_aux_coord(time_coord)
```

Comment:
Can use: `aux_coords_and_dims=[(fp_coord, 0), (time_coord, None)]` to get the same effect.

Reply:
Done.